apcd-cms/src/apps/view_users/templates/view_users.html

```diff
@@ -14,22 +14,35 @@ <h1>View Users</h1>
   <hr />
   <p style="margin-bottom: 30px">View submitted users.</p>
   <hr />
-  <div class="filter-container">
+  <div class="filter-container" style="display: flex; gap: 10px;">
     <div class="filter-content">
       <span><b>View Users by Organization: </b></span>
       <select id="organizationFilter" class="organization-filter" onchange="filterTableByOrganization()">
```

Comment:
```suggestion
      <select id="organizationFilter" class="status-filter" onchange="filterTableByOrganization()">
```
Looks like the class `organization-filter` doesn't exist, this class should work for matching other filters on the website.